src/app/components/home/home.component.html

```diff
@@ -0,0 +1,30 @@
+<app-action-card-component
+  titleColor="red"
+  title="Card for Organizing Activities"  
+  addActivityText="Add activity"
+  deleteText="Delete?"
+  [cardContentRow]="[{
```

Comment:
you can add more objects here and put content for other rows. In this case you will be able to add logic for <hr/>